social-network/src/ArticlePage.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import missedArticles from './missed-articles.json';
+import yourArticles from './your-articles.json';
+import Article from './Article';
+import MissedArticle from './MissedArticle';
+
+class ArticlePage extends Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            myMissedArticles: missedArticles,
```

Comment:
While it's nice to have these in state because they can be mutated if needed, they can also be used directly w/o being instantiated in state.